.travis.yml

```diff
@@ -1,54 +1,39 @@
-dist: trusty
-sudo: false
 language: php
-
-cache:
-  directories:
-    - $HOME/.composer/cache
-
 php:
-  - 7.1
-  - 7.2
-  - nightly
-
-before_install:
-  - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
-
-install:
-  - travis_retry composer update -n --prefer-dist
+    - 7.1
+    - 7.2
+    - 7.3
+    - nightly
 
-script: ./vendor/bin/phpunit
+env:
+    - DEPENDENCIES=highest
+    - DEPENDENCIES=lowest
 
-jobs:
-  allow_failures:
-    - php: nightly
-    - env: DEV_DEPENDENCIES
-
-  include:
-    - stage: Test
-      env: LOWEST_DEPENDENCIES
-      install:
-        - travis_retry composer update -n --prefer-dist --prefer-lowest
+install: |
+    if [ "$DEPENDENCIES" = "lowest" ]; then
+        composer update -n --prefer-stable --prefer-dist --prefer-lowest
+    else
+        composer update -n --prefer-stable --prefer-dist
+    fi
 
-    - stage: Test
-      env: LOWEST_DEPENDENCIES
-      php: 7.2
-      install:
-        - travis_retry composer update -n --prefer-dist --prefer-lowest
+script: make test
 
-    - stage: Test
-      env: LOWEST_DEPENDENCIES
-      php: nightly
-      install:
-        - travis_retry composer update -n --prefer-dist --prefer-lowest
+stages:
+    - Code quality
+    - Test
 
-    - stage: Test
-      env: DEV_DEPENDENCIES
-      php: nightly
-      install:
-        - composer config minimum-stability dev
-        - travis_retry composer update -n --prefer-dist
-
-    - stage: Coding standard
-      script:
-        - ./vendor/bin/phpcs
+jobs:
+    allow_failures:
+        - php: nightly
+
+    include:
+        - stage: Code quality
+          name: Lint
+          php: 7.3
```

Comment:
Tohle by se mělo pouštět na nejni

Reply:
Proč? Pouští se pak testy, tak případně failne tam kde je chyba ...

Reply:
Třeba u sniffu dostáváš pak jiný výsledky pro php 7.2 a 7.1 s tím, že to pak není BC. Třeba na 7.2 to bude vyzadovat `object` type hint, na 7.1 je to syntax error